docs/UserGuide.md

```diff
@@ -157,120 +112,107 @@ This command is used to add an employee to the address book. Let's examine how w
 
-* **What It Does:**
-  * Adds a tag to an employee record, useful for classifying employees like 'remote', 'contract', etc.
+Here are the potential error messages you may receive and here's how to fix them:
 
-* **Format:**
-  * `add-tag EMPLOYEE_LIST_INDEX TAG`
-* **Examples:**
+| Error message | Why it happens | Fix |
+|---------------|----------------|-----|
+| `Invalid command format!` | The command you input does not follow the specfied format | Ensure the command you entered follows the following format: `delete-tag INDEX t/TAG...`, replacing INDEX with the index of the employee currently, `TAG` is the name of the tag to be deleted and `t/TAG...` representing one or more tags. |
+| `At least one tag must be provided` | No tags were provided | Add tags to the command in the command box. Note that the tags must have a t/ [prefix](#glossary). For example, to remove the tag full-time, use `t/full-time` |
+| `The person index provided is invalid` | The index specified does not refer to any employee | Double check if the index appears in the employee list. Alternatively, use [list]() or any [find commands]() to locate the employee in the employee list. Afterwards, use the correct employee index in the `delete-tag` command. |
+| `Some of the tags are not found on this employee.` | The employee does not have some of the tags you are trying to delete | Remove the tags not found on the employee from the input command. For example, for an employee without the tag full-time, the command `delete-tag 2 t/full-time t/remote` does not work. Instead try `delete-tag 2 t/remote`.|
+| `Tags names only allows alphanumeric characters, spaces, and dashes.` | The tags input contains illegal characters | Remove the illegal characters from the input. |
 
-| S/N | Command information                                                                                                                                                                                                                        |
-|-----|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| 1   | **Command:** `add-tag 3 t/remote` <br><br> **Output:** Adds the tag “remote” to the employee whose index is 3. For instance, if “Paolo Maldini” is the employee whose index is 3, Maldini’s record will now contain the tag “remote”.        |
-| 2   | **Command:** `add-tag 14 t/temp t/contract` <br><br> **Output:** Adds the tag “temp” “contract” to the employee whose index is 14. For instance, if “Thierry Henry” is the employee whose index is 14, Henry’s record will now contain the tags “temp” and “contract”  |
+#### Editing the name, phone number, email address, home address or tags of employees : `edit`
+1. Get the [index](#glossary) of the employee under the [employee list](#glossary).
+
+<box type="info" seamless>
```

Comment:
It seems like the indentation here is a bit weird, perhaps can put it as point form?

Reply:
i am thinking of using an info box instead of a bullet, which does not render in markdown, but should work for the website
thanks